src/frontend/src/session.rs

```diff
@@ -382,6 +382,40 @@ impl FrontendEnv {
         });
         join_handles.push(join_handle);
 
+        // Heartbeat
+        let heartbeat_worker_node_manager = worker_node_manager.clone();
+        let join_handle = tokio::spawn(async move {
+            let duration = std::cmp::max(
+                Duration::from_secs(meta_config.max_heartbeat_interval_secs as _) / 10,
```

Comment:
How do we get `/ 10`? The serving node will be unmasked after the duration. Is this expected?

Reply:
Replace it with `mask_worker_temporary_secs` which is 30s by default. This 30s is first introduced in this PR.#10328